backend/app.js

```diff
@@ -63,6 +63,18 @@ app.get("/validation", (req, res) => {
   }
 });
 
+app.get("/nameValidation", (req, res) => {
+  let name = req.query.name;
+  console.log(name);
+  if (name != "") {
```

Comment:
```suggestion
  if (name !== "") {
```

https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons